src/gss_creds.c

```diff
@@ -520,7 +521,7 @@ uint32_t gssntlm_acquire_cred(uint32_t *minor_status,
                               gss_OID_set *actual_mechs,
                               uint32_t *time_rec)
 {
-    return gssntlm_acquire_cred_from(minor_status,
+    return gssntlm_acquire_cred_from(minor_status, NULL,
```

Comment:
Nit: adding these on the same line doesn't fit the local style of one argument per line.

Reply:
fixed locally, will push later when full reviews are in